File: src/Employee.java
public class Employee {

    private String name;
    private String lastName;
    private String birthDate;
    private String startDate;
    private String RFC;
    private String CURP;
    private double salary;
    private String rol;
    private String schedule;


    public Employee(String name, String lastName, String birthDate, String startDate, String RFC, String CURP, double salary, String rol, String schedule) {
        this.name = name;
        this.lastName = lastName;
        this.birthDate = birthDate;
        this.startDate = startDate;
        this.RFC = RFC;
        this.CURP = CURP;
        this.salary = salary;
        this.rol = rol;
        this.schedule = schedule;
    }



    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public String getBirthDate() {
        return birthDate;
    }

    public void setBirthDate(String birthDate) {
        this.birthDate = birthDate;
    }

    public String getStartDate() {
        return startDate;
    }

    public void setStartDate(String startDate) {
        this.startDate = startDate;
    }

    public String getRFC() {
        return RFC;
    }

    public void setRFC(String RFC) {
        this.RFC = RFC;
    }

    public String getCURP() {
        return CURP;
    }

    public void setCURP(String CURP) {
        this.CURP = CURP;
    }

    public double getSalary() {
        return salary;
    }

    public void setSalary(double salary) {
        this.salary = salary;
    }

    public String getRol() {
        return rol;
    }

    public void setRol(String rol) {
        this.rol = rol;
    }

    public String getSchedule() {
        return schedule;
    }

    public void setSchedule(String schedule) {
        this.schedule = schedule;
    }
}

File: src/Employee.py
from Zoo import Zoo


class Employee:
    __name = ""
    __last_name = ""
    __birth_date = ""
    __start_date = ""
    __rfc = ""
    __curp: object = ""
    __salary = 0
    __rol = ""
    __schedule = ""

    def __init__(self, name, last_name, birth_date, start_date, rfc, curp, salary, rol, schedule):
        self.__name = name
        self.__last_name = last_name
        self.__birth_date = birth_date
        self.__start_date = start_date
        self.__rfc = rfc
        self.__curp = curp
        self.__salary = salary
        self.__rol = rol
        self.__schedule = schedule
        self.zoo = Zoo()
        self.zoo.add_employee(self)

    def get_name(self):
        return self.__name

    def set_name(self, name):
        self.__name = name

    def get_last_name(self):
        return self.__last_name

    def set_last_name(self, last_name):
        self.__last_name = last_name

    def get_birth_date(self):
        return self.__birth_date

    def set_birth_date(self, birth_date):
        self.__birth_date = birth_date

    def get_start_date(self):
        return self.__start_date

    def set_start_date(self, start_date):
        self.__start_date = start_date

    def get_rfc(self):
        return self.__rfc

    def set_rfc(self, rfc):
        self.__rfc = rfc

    def get_curp(self):
        return self.__curp

    def set_curp(self, curp):
        self.__curp = curp

    def get_salary(self):
        return self.__salary

    def set_salary(self, salary):
        self.__salary = salary

    def get_rol(self):
        return self.__rol

    def set_rol(self, rol):
        self.__rol = rol

    def get_schedule(self):
        return self.__schedule

    def set_schedule(self, schedule):
        self.__schedule = schedule
